website/docs/error-reference.md

```diff
@@ -2295,12 +2295,6 @@ Some alternatives:
 
 It's an error to use `T.attached_class` to describe the type of method parameters. See the [T.attached_class](attached-class.md#tattached_class-as-an-argument) documentation for a more thorough description of why this is.
 
-## 5059
```

Comment:
This error will still show up for people on old versions, so let's keep it around.

We can add the same note about this functionality having been deleted in the most recent version.